Adding screenshots and APK link to README.md

raydelto opened the pull request:
Adding screenshots and APK link to README.md as suggested on issue https://github.com/chukong/SampleGame-FantasyWarrior3D/issues/214

Commit:
Adding screenshots and APK link to README.md

ahcox commented:
Issue did *not* request an APK link. 
Please don't merge it with a link to a binary.

The pictures are pretty though.

Commit:
Removing APK link

raydelto commented:
I've removed the APK link from the Pull Request.

ahcox reviewed (changes requested):
Put the images on an independent host we know will stay around or include them in the repo and link to them in here.

Commit:
Storing the screenshots in the repo

raydelto committed:
Fixing title markups on README.md

Adding space characters between the pound symbol(#) and the actual title. The lack of space was preventing the README.MD file from being rendered properly.

raydelto committed:
Fixing typo en README.md

Fixing a typo on the word `project` inside the platforms section

raydelto commented:
Images are now stored in the repo.
